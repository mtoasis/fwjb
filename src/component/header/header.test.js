import React from 'react'
import { shallow } from 'enzyme'
import Header from './header.component'

describe('Header component', () => {

    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Header />);
    })

    it('should render Header component', () => {
        expect(wrapper).toMatchSnapshot();
    })
})

