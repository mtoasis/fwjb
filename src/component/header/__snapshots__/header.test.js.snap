// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header component 1`] = `
<styled.div>
  <styled.div>
    <Connect(SearchBar) />
  </styled.div>
  <styled.div>
    <Styled(Link)
      to="/"
    >
      Main
    </Styled(Link)>
    <Styled(Link)
      to="/signin"
    >
      SignIn
    </Styled(Link)>
    <Styled(Link)
      to="/user/1"
    >
      Profile
    </Styled(Link)>
  </styled.div>
</styled.div>
`;
